crates/symbolicator-service/src/services/symbolication/js.rs

```diff
@@ -137,30 +141,38 @@ async fn symbolicate_js_frame(
 
     let smcache = match &module.smcache {
         Ok(smcache) => smcache,
-        Err(CacheError::Malformed(_)) => return Err(JsFrameStatus::MalformedSourcemap),
-        Err(_) => return Err(JsFrameStatus::MissingSourcemap),
+        Err(CacheError::Malformed(_)) => return Err(JsModuleErrorKind::MalformedSourcemap),
+        Err(_) => return Err(JsModuleErrorKind::MissingSourcemap),
     };
 
-    let mut frame = SymbolicatedJsFrame {
-        status: JsFrameStatus::Symbolicated,
-        raw: raw_frame.clone(),
-    };
+    let mut frame = raw_frame.clone();
 
+    // NOTE: The monolith processor creates a `EventError.JS_NO_COLUMN` error when `(Some(line), _)`
```

Comment:
We raise a `InvalidLocation` error with `col: None`, so we can detect this case with the errors we collect now.